src/main/webapp/WEB-INF/jsp/content/letter/edit.jsp

```diff
@@ -22,18 +22,6 @@
                 <div class="col s12">
                     <label><fmt:message key="allophones" /></label><br />
```

Comment:
@Shubham-Vishwakarma Every line of code related to allophones can be deleted, which would be the whole `<div class="row">` code block in this case.

Reply:
Done